dspec_tester.d

```diff
@@ -24,21 +24,82 @@ struct Config {
     string dmdBinPath = "dmd";
     bool printLineNumbers; // whether line numbers should be shown on errors
 }
-Config config;
+shared Config config;
```

Comment:
A cleaner solution would be:
```D
static immutable(Config)* config;
```

That way you can initialize the config and runtime and afterwards treat it as immutable from the parallel section of the code.